### ISSUE TITLE
Missing pipeline name in logging

### ISSUE THREAD

**Issue by pb-cdunn** [PacificBiosciences/pbsmrtpipe#17]
When I run `pbtestkit-runner testkit.cfg` in testkit-data/dev_01, the name of the top-level pipeline appears in *none* of the output. I know this because I've copied the whole directory and renamed the top-level pipeline, all else the same.

**Comment by mpkocher** [comment]
It's not the pipeline name, it's the test id. 

The test id should be defined in the testkit.cfg file. Currently, it's optional and (I believe) it will use the directory name. The test id is used by SIV to track integration tests.

```
(mk_pbsmrtpipe_test)dev_01 $> cat testkit.cfg 
[pbsmrtpipe:pipeline]

id = dev_01
description = Dev example
author = mkocher

# debug mode
debug = True

pipeline_xml = workflow_id.xml
preset_xml = preset.xml

[entry_points]
e_01 = input.txt


[tests]
# Tests can be loaded from any python module
# specifically, Any TestBase subclass in pbsmrtpipe.teskit.core.test_zero will be loaded
pbsmrtpipe.testkit.core = test_zero, test_resources, test_datastore
```

**Comment by pb-cdunn** [comment]
No, I'm looking for the pipeline name. It is in the workflow xml:

    4     
    5     <import-template id="pbsmrtpipe.pipelines.pipe_falcon" />

And that refers to the registered pipeline. It does not appear in any log.

**Comment by mpkocher** [comment]
Fixed in 0.20.3
